src/app/services/services.module.ts

```diff
@@ -146,15 +146,12 @@ export function initFactory(): Function {
     return async () => {
         await (storageService as HtmlStorageService).init();
 
-        if (process.env.ENV !== 'production' || platformUtilsService.isSelfHost()) {
+        const urls = process.env.URLS as Urls;
+        urls.base ??= window.location.origin;
+        if (platformUtilsService.isSelfHost()) {
```

Comment:
Do we still need this conditional with the change? Since we get the urls from `webpack` we should be able to reliably set urls to an empty object which would only resolve base.

Reply:
I didn't want to mess with our builds but I decided it was a good idea.